README.md

```diff
@@ -1,13 +1,14 @@
+[![RG3D](pics/logo2.svg)](https://rg3d.rs/)
+
+# Rust Game 3D
```

Comment:
It should be **R**ust **G**ame engine **3D**, I think it is less confusing than Rust Game - which looks like it is game, not engine.

Reply:
Actually, I just realized that I can fix it by myself after merge